config/routes.rb

```diff
@@ -30,9 +30,12 @@
       resources :job_applications, only: %i[index show destroy] do
         resources :build, only: %i[show update], controller: "job_applications/build"
         resources :employments, only: %i[new create edit update destroy], controller: "job_applications/employments"
-        resources :qualifications, only: %i[new create edit update destroy], controller: "job_applications/qualifications" do
-          get :select_category, on: :collection
-          post :submit_category, on: :collection
+        resources :qualifications, only: %i[new create edit update], controller: "job_applications/qualifications" do
+          collection do
+            delete :destroy, as: :destroy
```

Comment:
What's the reason for the destroy route not being in the array on line 33?

Reply:
That would create a route for destroying a single thing and expect a single id; this one will take as a param a list of records to destroy.